_translations/en.yml

```diff
@@ -193,6 +193,7 @@ en:
     walletbitgo: "BitGo is a high-security multi-sig wallet, which protects your bitcoin from theft and loss. You maintain full custody; BitGo cannot spend or freeze funds. BitGo wallets are easy to use and offer advanced security features such as spending limits and multi-user access."
     walletgreenaddress: "GreenAddress is a user-friendly multi-signature wallet with improved security and privacy. At no time are your keys server side, even encrypted. For security reasons, you should always use 2FA and the browser extension or Android App."
     wallettrezor: "Trezor is a hardware wallet providing a high level of security without sacrificing convenience. Unlike cold storage, Trezor is able to sign transactions while connected to an online device. That means spending bitcoins is secure even when using a compromised computer."
+    walletcoldcard: "Coldcard is a easy to use, ultra-secure, opensource and affordable hardware wallet.  NO app required.  Easy back-up! MicroSD card slot for encrypted backups. Your private key is stored in a dedicated real crypto security chip. Open source software design runs Micropython and you can change it."
```

Comment:
As mentioned in other reviews, this should be changed to be consistent with the style of other descriptions.

I would recommend the following:

> The Coldcard is an easy to use, ultra-secure, open-source, and affordable hardware wallet with no app required to use. Encrypted backups are easy to do with the built in MicroSD card slot. Your private key is stored in a dedicated real crypto security chip. The open-source software design runs MicroPython and can be changed by you.

Reply:
👍